zipkin-server/src/main/java/zipkin2/server/internal/elasticsearch/ZipkinElasticsearchStorageConfiguration.java

```diff
@@ -163,9 +163,7 @@ ScheduledExecutorService dynamicCredentialsScheduledExecutorService(
     @Value("${" + CREDENTIALS_REFRESH_INTERVAL + "}") Integer credentialsRefreshInterval,
     @Qualifier(QUALIFIER) BasicCredentials basicCredentials) throws IOException {
     ScheduledExecutorService ses = Executors.newSingleThreadScheduledExecutor(
-      new ThreadFactoryBuilder()
-        .setNameFormat("LoadElasticSearchCredentials-%d")
-        .build());
+      new NamedThreadFactory("zipkin-load-es-credentials-pool"));
```

Comment:
Minor nit: drop the pool suffix perhaps.

Reply:
done